app/controllers/concerns/foreman/controller/smart_proxy_auth.rb

```diff
@@ -68,7 +68,7 @@ def auth_smart_proxy(proxies = SmartProxy.all, require_cert = true)
           # If the client sent certificate contains a subject or sans, use them for request_hosts, else fall back to the dn set in the request environment
           request_hosts = []
           if certificate
-            if certificate.subject_alternative_names
+            if ! certificate.subject_alternative_names.empty?
```

Comment:
There's a space in between `!` and the variable that rubocop (our Ruby linter) doesn't like :smile: